tests/sentry/hybrid_cloud/test_organizationmapping.py

```diff
@@ -103,9 +58,72 @@ def test_update(self):
         org_mapping = OrganizationMapping.objects.get(organization_id=self.organization.id)
         assert org_mapping.customer_id == "test"
         assert org_mapping.name == "new name!"
+        assert org_mapping.slug == self.organization.slug
+        assert org_mapping.status == self.organization.status
 
         organization_mapping_service.update(
             organization_id=self.organization.id, update=RpcOrganizationMappingUpdate()
         )
         # Does not overwrite with empty value.
         assert org_mapping.name == "new name!"
+
+    def test_upsert__create_if_not_found(self):
+        self.organization = self.create_organization(
+            name="test name",
+            slug="foobar",
+        )
+
+        fixture_org_mapping = OrganizationMapping.objects.get(organization_id=self.organization.id)
+        fixture_org_mapping.delete()
+
+        assert len(OrganizationMapping.objects.filter(organization_id=self.organization.id)) == 0
```

Comment:
Django also has `QuerySet.count()` and `QuerySet.exists()` if you want to check for records existing without asserting the contents of the record.